.circleci/config.yml

```diff
@@ -1,9 +1,8 @@
 version: 2.1
-
 orbs:
-  secrethub: secrethub/cli@1.1.0
+  keeper: gravitee-io/keeper@0.6.1
   gh: circleci/github-cli@2.0.0
-
+  
```

Comment:
We need to remove the extra space

Reply:
@sursak123 can you please update your IDE to not add these extra spaces and remove them from your commit?